.github/scripts/label_promoted_commits.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from github import Github
+import argparse
+import re
+import sys
+import os
+
+try:
+    github_token = os.environ["GITHUB_TOKEN"]
+except KeyError:
+    print("Please set the 'GITHUB_TOKEN' environment variable")
+    sys.exit(1)
+
+
+def parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--repository', type=str, required=True,
+                        help='Github repository name (e.g., scylladb/scylladb)')
+    parser.add_argument('--commit_before_merge', type=str, required=True, help='Git commit ID to start labeling from ('
+                                                                               'newest commit).')
+    parser.add_argument('--commit_after_merge', type=str, required=True,
+                        help='Git commit ID to end labeling at (oldest '
+                             'commit, exclusive).')
+    parser.add_argument('--update_issue', type=bool, default=False, help='Set True to update issues when backport was '
+                                                                         'done')
+    parser.add_argument('--label', type=str, required=True, help='Label to use')
+    return parser.parse_args()
+
+
+def main():
+    args = parser()
+    pr_pattern = re.compile(r'Closes .*#([0-9]+)')
```

Comment:
I see merged PRs in scylla without this pattern.
E.g: https://github.com/scylladb/scylladb/pull/17506

Reply:
This pattern is only applied to single-patch PRs.
Multi-patch PRs are merged instead of rebased, and github closes the PR due to this merge, and doesn't need the Closes.

Reply:
THe pattern is in the merge commit we create - https://github.com/scylladb/scylladb/commit/b4cef638ef4b9b9765de704d919b77da66078837

Reply:
We don't update the PR wit this pattern